### PR TITLE
feat: initial version key_vault_key module

### PR DESCRIPTION
**Pull request by roolrd**
modified:   README.md
	new file:   main.tf
	new file:   outputs.tf
	new file:   variables.tf
	new file:   versions.tf

### CODE REVIEW

**Review by owlleg6 — APPROVED** [review]
LGMT

File: README.md [hunk 1]
```diff
@@ -1,10 +1,54 @@
...
+
+## Outputs
+
+| Name                                                           | Description            |
+|----------------------------------------------------------------|------------------------|
+| <a name="output_key_name"></a> [key\_name](#output\_key\_name) | The Key Vault Key name |
+| <a name="output_key_id"></a> [key\_id](#output\_key\_id)       | The Key Vault Key ID   |
 <!-- END_TF_DOCS -->
 
 ## License
```
Comment (Dmytrovel) [inline README.md:71]: fix licence path

File: README.md [hunk 2]
```diff
@@ -1,10 +1,54 @@
...
 
 ## Usage
 
 <!-- BEGIN_TF_DOCS -->
+## Requirements
 
+| Name                                                                      | Version   |
+|---------------------------------------------------------------------------|-----------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.0.0  |
+| <a name="requirement_azurerm"></a> [azurerm](#requirement\_azurerm)       | >= 3.46.0 |
```
Comment (Dmytrovel) [inline README.md]: change to 3.60.0

File: README.md [hunk 3]
```diff
@@ -1,10 +1,54 @@
...
+
+## Providers
+
+| Name                                                          | Version |
+|---------------------------------------------------------------|---------|
+| <a name="provider_azurerm"></a> [azurerm](#provider\_azurerm) | 3.46.0  |
```
Comment (Dmytrovel) [inline README.md]: change to 3.60.0

File: README.md (hunk 1 above)
Comment (roolrd) [reply README.md:71]: fixed

File: README.md (hunk 2 above)
Comment (roolrd) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (roolrd) [reply README.md]: changed

**Review by Dmytrovel — APPROVED** [review]

**Pull request merged**